### PR TITLE
Updated shrinkwrap

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Need the latest version of Fabric Core so am updating the shrinkwrap.

#### Focus areas to test

(optional)

### CODE REVIEW

File: common/config/rush/npm-shrinkwrap.json [hunk 1]
```diff
@@ -4,18 +4,13 @@
...
-      "version": "3.0.9",
-      "resolved": "https://registry.npmjs.org/@hypnosphi/fuse.js/-/fuse.js-3.0.9.tgz",
-      "integrity": "sha1-6pn2EhtKjwZbTHH4VZXbJxRJiAc="
-    },
     "@microsoft/api-extractor": {
-      "version": "4.3.4",
-      "resolved": "https://registry.npmjs.org/@microsoft/api-extractor/-/api-extractor-4.3.4.tgz",
-      "integrity": "sha1-YLgRx/KPIVPpM2qS26p06RWiA/Q=",
+      "version": "4.3.7",
+      "resolved": "https://onedrive.pkgs.visualstudio.com/_packaging/odsp-npm/npm/registry/@microsoft/api-extractor/-/api-extractor-4.3.7.tgz",
```
Comment (lynamemi) [inline common/config/rush/npm-shrinkwrap.json]: I'm not sure why the "resolved" base urls all changed. I ran the command from rush v.4.1.0 so I'm not sure what other factors on my end could contribute to this change.

Comment (Jahnp) [reply common/config/rush/npm-shrinkwrap.json]: This might be due to your user's.npmrc config--did you modify that recently (e.g. for doing product work)?

Comment (lynamemi) [reply common/config/rush/npm-shrinkwrap.json]: ahhhh that could be! I did modify that recently for prod work. I'll take a look.

Comment (lynamemi) [reply common/config/rush/npm-shrinkwrap.json]: Yep, that was the culprit!

**Comment by lynamemi** [comment]
I have another screener mystery. Not sure where the command bar changes is coming from. I did pull from master. And I can't find anything suspicious in command bar in my repo.

File: common/config/rush/npm-shrinkwrap.json [hunk 2]
```diff
@@ -8638,22 +9012,22 @@
       "integrity": "sha1-EEEktsYCxnlogaBCVB0220OlJk4="
     },
     "office-ui-fabric-core": {
-      "version": "9.3.0",
-      "resolved": "https://registry.npmjs.org/office-ui-fabric-core/-/office-ui-fabric-core-9.3.0.tgz",
-      "integrity": "sha1-FE19kSUBS9+lf41fXmptgl2EJ0I="
+      "version": "9.4.0",
```
Comment (mikewheaton) [inline common/config/rush/npm-shrinkwrap.json]: 🎉

**Comment by lynamemi** [comment]
Now I'm picking up those weird people picker changes in screener that were haunting us earlier...

**Comment by dzearing** [comment]
I opened another PR here.

**Pull request closed without merging**